mandatory/3-playing-computer.js

```diff
@@ -8,11 +8,11 @@
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  3. What is printed to the console? Awnser: 2,6,4,9,6,13,8
+  4. How many times is "f1" called?  Awnser: 2 times (2 odd number 1 3)
+  5. How many times is "f2" called?  Awnser: 3 times (3 even number 0 2 4)
+  6. What value does the "a" parameter take in the first "f1" call?  Awnser: 8
+  7. What is the value of the "a" outer variable when "f1" is called for the first time? Awnser: 6
```

Comment:
The outer variable `a` is initialised with a value of 6.

This value is then incremented by 1 for each iteration of the `for` loop.

Therefore, just before we call `f1` for the first time then `i` must have a value of 1: therefore `a` will have been incremented by 1 twice already. Therefore `a` will have a value of 8.